README.md

```diff
@@ -43,7 +43,8 @@ And then the _SDK Location_ section. It will show you the NDK path, or the optio
 
 ### Building
 
-- Install dependencies `yarn bootstrap`
+- `yarn install`
```

Comment:
yarn install is not necessary here. Is it because you are using yarn v3 maybe?

Reply:
I think it's still needed?

I just saw it and assumed the direction should be more explicit. 

Feel free to close if not needed.